modules/environment-roles/templates/jenkins_lambda_policy.json.tpl

```diff
@@ -25,6 +25,7 @@
         "arn:aws:lambda:eu-west-2:${account_id}:function:tdr-download-files-${environment}",
         "arn:aws:lambda:eu-west-2:${account_id}:function:tdr-database-migrations-${environment}",
         "arn:aws:lambda:eu-west-2:${account_id}:function:tdr-create-keycloak-db-user-${environment}",
+        "arn:aws:lambda:eu-west-2:${account_id}:function:tdr-create-keycloak-db-user-new-${environment}",
```

Comment:
Assume the `new` is in the name to prevent breaking changes?

Reply:
Yep, it's on my list of things to do but it's referenced in quite a few places so it needs doing carefully.